target/linux/siflower/sf21/config-6.12

```diff
@@ -49,6 +53,7 @@ CONFIG_CRYPTO_LIB_UTILS=y
 CONFIG_CRYPTO_ZSTD=y
 CONFIG_DEBUG_BUGVERBOSE=y
 CONFIG_DEBUG_INFO=y
+# CONFIG_KERNEL_UNCOMPRESSED is not set
```

Comment:
Were you manually editing this file? Please use `make kernel_menuconfig` instead.